src/MICore/LaunchOptions.xsd

```diff
@@ -228,6 +228,11 @@
               <xs:documentation>Path to the MI Debugger executable. If only the executable name is provided, it will attempt to search $PATH for the location of the executable.</xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="MIDebuggerArgs" type="xs:string" use="optional">
```

Comment:
Add to the "ServerOptions" too.

Reply:
Am I supposed to add ServerOptions to the xml? 

DebugServer, DebugServerArgs, ServerStarted, FilterStdout, FilterStderr, and ServerLaunchTimeout are already in LocalLaunchOptions

Reply:
ServerOptions is in the xml. It is used in attach scenarios. It has an "MIDebuggerPath" field. It just needs an instance of your new MIDebuggerArgs field too. Then the field needs to be merge into the final attach launch options (as appropriate) in CreateForAttachRequest  (see below).

Reply:
Ah... I totally missed that field.